test/delivery_methods/ios_test.rb

```diff
@@ -18,6 +18,9 @@ def push(apn)
       @deliveries.push(apn)
       @response
     end
+
+    def close
+    end
```

Comment:
What is this for?

Reply:
We are calling `connection.close` in the `Noticed::DelvieryMethods::Ios#deliver`.  This is a default mock to make sure the test do not error closing the connection.

Reply:
Ah yes, makes sense! I wasn't paying attention that it was in the Fake Connection pool. 👍